ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-path-selection.j2

```diff
@@ -21,6 +21,16 @@ router path-selection
 {%         if path_group.ipsec_profile is arista.avd.defined %}
       ipsec profile {{ path_group.ipsec_profile }}
 {%         endif %}
+{%         if path_group.keep_alive is arista.avd.defined %}
+{%             set interval_cli = "" %}
+{%             if path_group.keep_alive.auto is arista.avd.defined(true) %}
+{%                 set interval_cli = "auto" %}
+{%             elif path_group.keep_alive.interval is arista.avd.defined and
+                    path_group.keep_alive.failure_threshold is arista.avd.defined %}
+{%                 set interval_cli = path_group.keep_alive.interval ~ " milliseconds failure-threshold " ~ path_group.keep_alive.failure_threshold ~ " intervals" %}
+{%             endif %}
+      keepalive interval {{ interval_cli }}
+{%         endif %}
```

Comment:
This will configure `keepalive interval ` (notice the space at the end) if no options are given for interval.
I think we should change it to:
```suggestion
{%         if path_group.keep_alive.auto is arista.avd.defined(true) %}
      keepalive interval auto
{%         elif path_group.keep_alive.interval is arista.avd.defined and
                path_group.keep_alive.failure_threshold is arista.avd.defined %}
      keepalive interval {{ path_group.keep_alive.interval }} milliseconds failure-threshold {{ path_group.keep_alive.failure_threshold }} intervals
{%         endif %}
```